### PR TITLE
Add bors config

### PR DESCRIPTION
**Pull request by Robert-Steiner**
enables bors

there are some more options we can set 
https://bors.tech/documentation/

Let me know if I should add/remove any of them

### CODE REVIEW

**Review by wilk10 — APPROVED** [review]
are you waiting for review/approval here as well?

**Comment by Robert-Steiner** [comment]
> are you waiting for review/approval here as well?

not right now 😊 
but you can review #625 and #640 if you like

**Comment by Robert-Steiner** [comment]
we will tackle this later
first we should check if bors is working together with 
- dependabot 
- protected branches

**Pull request closed without merging**
